emuu-client/src/components/UserAuthentication/newloginscreen.js

```diff
@@ -38,15 +38,15 @@ function Login() {
     const user = auth.currentUser;
     signInWithEmailAndPassword(auth, email, password)
       .then((userCredential) => {
-      console.log(userCredential)
+        console.log(userCredential);
```

Comment:
We should have local storage removed in unnecessary places after the next prototype. Also, I don't see A need to console log this.